### DISCUSSION TITLE
PCAP Rollover not working correctly.

### DISCUSSION THREAD

**Discussion by Inkman647** [Security-Onion-Solutions/securityonion#9225] [Unsupported Versions]
Hello, first of all going to mention that we're using Arkime instead of Stenographer, in a distributed environment.  Meaning we set our PCAP path to /nsm/arkimepcap to differentiate. These factors shouldn't effect this problem though.

The PCAP files are rolling over at 90% fine. I can see the filecount for the 23rd decreasing, and the filecount for today increasing.

However, there's several hundred files PCAP files (most 13 GB each) from the 15th/16th/17th that aren't being deleted. This should be way past our rollover point. The read/write permissions are the same. The date modified is the dates they were created.

Is there a way to mark a PCAP file to not be deleted in the SOConsole? Rather than downloading it somewhere. And if so is there a way for someone to have done that for hundreds of files on accident?

See picture attached. Shows the filecount of each date.  
![sketch1669558890781](https://user-images.githubusercontent.com/101215740/204140202-6d51325a-a8bb-4add-b26d-f3d0f92fc55c.png)

Thank you

**Comment by dougburks** [comment]
By default, we configure Stenographer to purge its own files. Since you've decided not to use Steno and instead use Arkime which we don't include or support, we're not able to assist you any further.
